Docs/src/rtss.md

```diff
@@ -494,17 +494,17 @@ In order to extend the system with your own shader effects you'll have to follow
 * Add shader files that will supply all the actual shader functions your SubRenderState needs. In order to support multiple shader languages, @ref OgreUnifiedShader are provided. These shaders should be placed in a resource location known to the resource manager.
 
 Implementing the SubRenderState requires overriding the pure methods of the base class.
-* Ogre::RTShader::SubRenderState::getType() should return unique string that identify the sub class implementation. That value is shared among all instances and can be stored in a static string variable. It uses to system to match between SubRenderState instance and the factory to should destroy it.
+* Ogre::RTShader::SubRenderState::getType() should return a unique string that identifies the sub class implementation. That value is shared among all instances and can be stored in a static string variable. It is used by the system to match between the SubRenderState instance and the factory that should destroy it.
```

Comment:
I had difficulties understanding the last sentence, hopefully I got the meaning right. Let me know otherwise.